azure-pipelines.yml

```diff
@@ -54,11 +54,11 @@ steps:
   displayName: Sign exe, dll and powershell files
   inputs:
     ConnectedServiceName: 'CodeSigning-APEX'
-    FolderPath: '$(System.DefaultWorkingDirectory)'
+    FolderPath: '$(System.DefaultWorkingDirectory)/ECMA2Yaml'
```

Comment:
Just to confirm. I see you also added `ECMA2Yaml` at the start of some paths. Does the duplication reflect the folder structure correctly?

Reply:
yes - https://github.com/docascode/ECMA2Yaml/tree/develop/ECMA2Yaml/ECMA2Yaml